src/Compilers/Core/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -534,6 +534,30 @@ static void Main()
 
         [Fact]
         [Trait(Traits.Environment, Traits.Environments.VSProductInstall)]
+        public void CompileErrorsCSWithAnalyzers()
+        {
+            Dictionary<string, string> files =
+                                   new Dictionary<string, string> {
+                                           { "hello.cs",
+@"using System;
+class Hello 
+{
+    static void Main()
+    { Console.WriteLine(""Hello, world."") }
+}"}};
+
+            var result = RunCommandLineCompiler(_csharpCompilerClientExecutable, "/reportanalyzer hello.cs", _tempDirectory, files);
+
+            // Should output errors, but not create output file.                  
+            Assert.Contains("Copyright (C) Microsoft Corporation. All rights reserved.", result.Output, StringComparison.Ordinal);
+            Assert.Contains("hello.cs(5,42): error CS1002: ; expected: Microsoft.CodeAnalysis.Diagnostics.CSharp.CSharpCompilerDiagnosticAnalyzer\r\n", result.Output, StringComparison.Ordinal);
```

Comment:
@AnthonyDGreen I should have added anthony. 

no nothing is approved from anyone. this PR is to see whether what I am doing is even right way to add argument to compiler.